### PR TITLE
Potential LBF typos found with spellcheck

### PR DESCRIPTION
**Pull request by mattleff**
See https://github.com/bountonw/translate-tooling/issues/14#issuecomment-2092114720

### CODE REVIEW

File: th/LBF/03_public/LBF13_th.md
Comment (bountonw) [inline th/LBF/03_public/LBF13_th.md:1]: Is there a galan in the Bible text? I'm with people and can't check. But it seems that this is an exception.

File: th/LBF/03_public/LBF36_th.md
Comment (bountonw) [inline th/LBF/03_public/LBF36_th.md:1]: What changed?

File: th/LBF/03_public/LBF47_th.md
Comment (bountonw) [inline th/LBF/03_public/LBF47_th.md:1]: พืชพันธุ์ is correct. This shouldn't have been flagged. พันธ์  is a word, but not in this context.

File: th/LBF/03_public/LBF13_th.md
Comment (bountonw) [reply th/LBF/03_public/LBF13_th.md:1]: I double checked. There is no galan. พระเยซูคริสตเจ้า

The addition of เจ้า at the end lengthens the word and removes the galan.

**Review by bountonw — COMMENTED** [review]
I have made inline comments on two of the files that shouldn't have been flagged. The other files are real errors and are updated correctly.

File: th/LBF/03_public/LBF36_th.md
Comment (mattleff) [reply th/LBF/03_public/LBF36_th.md:1]: image

`บัญญิต` -> `บัญญัติ`

File: th/LBF/03_public/LBF47_th.md
Comment (mattleff) [reply th/LBF/03_public/LBF47_th.md:1]: Fixed with 09a65ea.

File: th/LBF/03_public/LBF13_th.md
Comment (mattleff) [reply th/LBF/03_public/LBF13_th.md:1]: @bountonw Ok, there ended up being 4 places in LBF and 1 each in MB + PP that used the galan-version that I fixed to be the non-galan version: 09a65ea. I also pushed a PR to add the galan-version to forbidden terms: https://github.com/bountonw/translate-tooling/pull/32

**Comment by mattleff** [comment]
@bountonw I just removed all the พระเยซูคริสตเจ้า-related changes. Double check these are all good and I think we can ship this PR.

**Review by bountonw — APPROVED** [review]

**Comment by bountonw** [comment]
@mattleff You can merge this when you are ready.

**Pull request merged**